src/coreclr/jit/hwintrinsic.cpp

```diff
@@ -400,6 +400,61 @@ CorInfoType Compiler::getBaseJitTypeFromArgIfNeeded(NamedIntrinsic       intrins
     return simdBaseJitType;
 }
 
+//------------------------------------------------------------------------
+// gtIsBitwiseIntrinsic: Check if the given hwintrinsic is a specific type of bitwise intrinsic.
+//
+// Arguments:
+//    intrinsicId       -- id of the intrinsic function.
+//    oper              -- the bitwise intrinisic type
+//
+// Return Value:
+//    True if the input intrinsic matches with the given bitwise intrisic type.
+//
+bool Compiler::gtIsBitwiseIntrinsic(NamedIntrinsic intrinsicId, genTreeOps oper)
```

Comment:
I think we already have an `HWOperGet()` so you can do `node->HWOperGet() == GT_AND` for example

Reply:
I could only find `OperGet()`, which in this case will return `GT_HWIntrinsic`, while not its semantic meaning. Would you please point me to the method you were referring to, thanks!